scripts/ts-bin/csvtojsonlines_test.ts

```diff
@@ -0,0 +1,34 @@
+import {csvtojsonlines} from './csvtojsonlines_lib';
+import * as stream from 'stream';
+import {expect} from 'chai';
+import 'mocha';
+
+describe('Hello function', () => {
+  it('should return hello world', async function () {
+    let r = new stream.Readable();
```

Comment:
Use const (I think the linter will detect these)

Reply:
yeah, I'm a little surprised it didn't... and then I saw there is no tslint.json file... lets add that in a separate PR. :)